olive/engine/engine.py

```diff
@@ -65,15 +65,15 @@ def __init__(
         elif self._config.host is not None:
             self.host = self._config.host.create_system()
         else:
-            self.host = LocalSystem()
+            self.host = LocalSystem([Device.CPU])
```

Comment:
host itself doesn't use the accelerators actually, so it could be safely removed.